Tests/Shared.Specs/BasicEquivalencySpecs.cs

```diff
@@ -2581,7 +2581,10 @@ public void When_a_specific_mismatching_property_is_excluded_from_conversion_it_
             //-----------------------------------------------------------------------------------------------------------
             // Assert
             //-----------------------------------------------------------------------------------------------------------
-            act.Should().Throw<XunitException>().WithMessage("Expected*<1973-09-20>*\"1973-09-20\"*");
+            string exceptionMessage = act.Should().Throw<XunitException>().Which.Message;
```

Comment:
🙃 Can't the assertions be chained with `And`?

Reply:
We should eat our own dog food right? ;-).